js/views/FormPage.js

```diff
@@ -135,6 +162,49 @@ const FormPage = React.createClass ({
   },
 
   /* Methods */
+  filterForms(forms) {
+    let past = new Date()
+    past = past.setDate(past.getDate() - 3)
+    filteredForms = _.filter(forms, function(form){
+      let triggerTime = form.trigger
+      return triggerTime > past && triggerTime < new Date()
+    })
+    return filteredForms
+  },
+
+  sortForms(forms){
+    return _.sortBy(forms, 'trigger')
+  },
+
+  triggers(form, self, callback){
+    loadTriggers(form, self.props.survey, function(err, triggers){
```

Comment:
Good catch...I've switched to using the cached triggers. It certainly simplifies things a good bit. Thanks, @nathanathan.